lizzy_client/utils.py

```diff
@@ -1,11 +1,37 @@
-from collections import namedtuple
+import os
 import re
-import yaml
+from collections import namedtuple
+from urllib.error import URLError
+from urllib.parse import quote
+from urllib.request import urlopen
 
+import click
+import yaml
 
 StackReference = namedtuple('StackReference', 'name version')
 
 
+def read_parameter_file(parameter_file):  # copy pasted from Senza
+    paras = []
+
+    try:
+        url = (parameter_file if '://' in parameter_file
+               else 'file://{}'.format(quote(os.path.abspath(parameter_file))))
+
+        response = urlopen(url)
+    except URLError:
+        raise click.UsageError('Can\'t read parameter file "{}"'.format(parameter_file))
+
+    try:
+        cfg = yaml.safe_load(response.read())
```

Comment:
This approach is OK, but we are duplicating code/behaviour of Senza in the Lizzy-Client. The "correct" way would be adding this "parameters-file" to the API.